source/common/config/tls_context_json.cc

```diff
@@ -59,6 +59,10 @@ void TlsContextJson::translateCommonTlsContext(
   for (const auto& san : json_tls_context.getStringArray("verify_subject_alt_name", true)) {
     validation_context->add_verify_subject_alt_name(san);
   }
+  if (json_tls_context.hasObject("verify_certificate_expiration")) {
```

Comment:
Drive by comment: Let's please not add this feature to v1 config. Can we revert this change and the schema change? Thank you.